### PR TITLE
Added API2000 support for network resources

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added API2000 support for network resources

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [x] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

**Review by chebroluharika — APPROVED** [review]

File: libraries/resource_providers/api2000/c7000/fcoe_network_provider.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API2000
+    module C7000
+      # FCoENetwork API2000 C7000 provider
+      class FCoENetworkProvider < API1800::C7000::FCoENetworkProvider
```
Comment (AsisBagga) [inline libraries/resource_providers/api2000/c7000/fcoe_network_provider.rb]: should be OneviewCookbook::API1800::C7000::FCoENetworkProvider

File: libraries/resource_providers/api2000/synergy/ethernet_network_provider.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API2000
+    module Synergy
+      # EthernetNetworkProvider API2000 Synergy provider
+      class EthernetNetworkProvider < API1800::Synergy::EthernetNetworkProvider
```
Comment (AsisBagga) [inline libraries/resource_providers/api2000/synergy/ethernet_network_provider.rb]: please fix this, same as above

File: libraries/resource_providers/api2000/synergy/fcoe_network_provider.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API2000
+    module Synergy
+      # FCoENetwork API2000 Synergy provider
+      class FCoENetworkProvider < API1800::Synergy::FCoENetworkProvider
```
Comment (AsisBagga) [inline libraries/resource_providers/api2000/synergy/fcoe_network_provider.rb]: Same as above

File: libraries/resource_providers/api2000/c7000/fcoe_network_provider.rb (hunk 1 above)
Comment (VenkateshRavula) [reply libraries/resource_providers/api2000/c7000/fcoe_network_provider.rb]: done

File: libraries/resource_providers/api2000/synergy/ethernet_network_provider.rb (hunk 2 above)
Comment (VenkateshRavula) [reply libraries/resource_providers/api2000/synergy/ethernet_network_provider.rb]: done

File: libraries/resource_providers/api2000/synergy/fcoe_network_provider.rb (hunk 3 above)
Comment (VenkateshRavula) [reply libraries/resource_providers/api2000/synergy/fcoe_network_provider.rb]: done

**Review by AsisBagga — APPROVED** [review]
LGTM

**Pull request merged**